website/package.json

```diff
@@ -11,39 +11,14 @@
     "swizzle": "docusaurus swizzle",
     "clear": "docusaurus clear",
     "serve": "docusaurus serve",
-    "write-heading-ids": "ts-node scripts/write-heading-ids.ts",
+    "write-heading-ids": "ocular-doc-headers ../docs",
     "deploy": "docusaurus deploy"
   },
   "dependencies": {
-    "@algolia/autocomplete-js": "^1.8.3",
-    "@docusaurus/core": "^3.0.0",
-    "@docusaurus/preset-classic": "^3.0.0",
-    "@loaders.gl/i3s": "^3.3.1",
-    "@loaders.gl/las": "^3.3.1",
-    "@loaders.gl/obj": "^3.3.1",
-    "@loaders.gl/ply": "^3.3.1",
-    "@material-ui/core": "^4.10.2",
-    "@material-ui/icons": "^4.9.1",
-    "@material-ui/lab": "^4.0.0-alpha.57",
-    "@mdx-js/react": "^3.0.0",
-    "docusaurus-mdx-checker": "^3.0.0",
-    "d3-color": "^3.1.0",
-    "d3-request": "^1.0.6",
-    "mapbox-gl": "^2.0.0",
-    "maplibre-gl": "^2.4.0",
+    "@docusaurus/core": "^2.0.0",
```

Comment:
That is a very old version of docusaurus...

Reply:
One of our components is using an internal React hook that no longer exists. I will poke deeper.